peer-signer/README.md

```diff
@@ -0,0 +1,26 @@
+# Peer Signer
+This tool is intended to verify a potential candidate for testnet validator. Candidate needs to be suplied with `signer` program and random secret message generated by us.
+
+## Step one: signature generation
+To generate signature candidate needs a generated p2p address. For whole process to work they also need to have a node set up with up to date block creation and public p2p port, so it can be added as a peer. To generate signature:
+```console
+$ signer --message "secret message" --p2p-secret "some_path_to_p2p_secret/p2p_secret"
+Public key: 08011220c94d0593359ea30451f3aaf61040756c1bfdb7bb779489d4dafda00cce038c0a
+Signed message: 7ef2490075425bcd9fc77a994069183a21746f0be3561d464ae6a89e4b9bdaf26d9cb75c0b11325ae54e66c8ce8a9156a8b8e1d0120309fafe6971db53d6c104
+```
+
+## Step two: communication
+User needs to send the generated public key and signature to verifier. So in this case needs to send:
+```
+08011220c94d0593359ea30451f3aaf61040756c1bfdb7bb779489d4dafda00cce038c0a
+
+7ef2490075425bcd9fc77a994069183a21746f0be3561d464ae6a89e4b9bdaf26d9cb75c0b11325ae54e66c8ce8a9156a8b8e1d0120309fafe6971db53d6c104
+```
+
+## Step three: verification
+To verify you need a node setup, that you can send unsafe rpc calls to. Let's say we have a running node with rpc socket on `http://127.0.0.1:9933`. This node needs to be up to date with block creation. Also for this process to work we need to have candidate in our peers, so to make sure we connected, we can add it to bootnodes of our node. Now we call `verifier` to verify peer and the signature:
```

Comment:
fat nit: in markdown I usually recommend putting each sentence on a separate line: it does not influence the rendered view, but makes it easier for commenting on a particular sentence (instead of a whole paragraph).